packages/core/core/src/factories/apiFactory/apiProxyFactory.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import { applyContextForApi } from './../../utils/context';
+import { Logger } from './../../utils';
+import { createInstance } from './_utils';
+import { ApiProxyFactoryParams, ApiInstance, ProxyFactoryInstance } from './types';
+import { getConfig, createProxy } from './_proxyUtils';
+
+const apiProxyFactory = <ALL_SETTINGS, ALL_FUNCTIONS>(factoryParams: ApiProxyFactoryParams<ALL_SETTINGS, ALL_FUNCTIONS>): ProxyFactoryInstance => {
+  function createApiProxy (config: any, customApi: any = {}): ApiInstance {
+    const _config = getConfig({ context: this, factoryParams, givenConfig: config });
```

Comment:
```suggestion
    const _config = getConfig({ context: this, factoryParams, givenConfig: config });
```

get config of what?

Maybe `getIntegrationConfig` ?